apps/api/src/users/controllers/user.controller.ts

```diff
@@ -13,6 +13,12 @@ export class UserController {
     return this.userService.getUserProfilesByUserId(user.id);
   }
 
+  @Post('profiles/search')
+  @Header('Cache-Control', 'non')
```

Comment:
non -> none?

Reply:
Actually, should be `no-cache`: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control

Reply:
Replaced on 'no-cache' everywhere